### PR TITLE
Add `<img>` element

### PR DESCRIPTION
**Pull request by petele**
Used `compute_from:` otherwise `html.elements.img.decoding` pushed it into later support.

### CODE REVIEW

**Comment by foolip** [comment]
https://github.com/web-platform-dx/web-features/pull/1448 is a new draft for `srcset` and `sizes`. Can you exclude those keys from this feature?

**Comment by petele** [comment]
moved `isMap` and `useMap` to #1427 per request in that PR

File: features/img.yml [hunk 1]
```diff
@@ -1,8 +1,9 @@
-draft_date: 2024-06-05
 name: <img>
-description: The `<img>` element TODO.
+description: The `<img>` element adds an image into the document.
```
Comment (foolip) [inline features/img.yml:2]: @ddbeck can you review this description?

**Comment by foolip** [comment]
I took the liberty of removing `currentSrc` and will add it to https://github.com/web-platform-dx/web-features/pull/1448.

**Review by foolip — APPROVED** [review]

**Pull request merged**